example/config-win.sample

```diff
@@ -0,0 +1,11 @@
+{
+  "block_store_path" : "C:\\Users\\user\\AppData\\Local\\Temp\\block_store",
```

Comment:
Is it necessary to escape each \?

Reply:
Why ```user``` is hard coded here?

Reply:
Yes, since backslash is an escape character in JSON strings.